app/config/AppConfig.scala

```diff
@@ -60,5 +60,6 @@ class AppConfigImpl @Inject()(config: ServicesConfig, configuration: Configurati
 
 trait FixedConfig {
   // Minimum tax year for MTD
-  val minimumTaxYear = 2019
+
+  val minimumTaxYear = 2022
```

Comment:
You need to move the minimumTaxYear from fixed config to the overridable appconfig section.